Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildUnityProjectExporter.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using Microsoft.Build.Unity.ProjectGeneration.Exporters;
+using System;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.Build.Unity.ProjectGeneration
+{
+    /// <summary>
+    /// This class contains the export logic given info about the current Unity project.
+    /// </summary>
+    public static class MSBuildUnityProjectExporter
+    {
+        /// <summary>
+        /// Exports the core Unity props file.
+        /// </summary>
+        /// <param name="exporter">The overal exporter to use for creating exporters.</param>
+        /// <param name="platform">The compilation platform to export.</param>
+        /// <param name="inEditor">Whether to export the inEditor version.</param>
+        public static void ExportCoreUnityPropFile(IUnityProjectExporter exporter, CompilationPlatformInfo platform, bool inEditor)
+        {
+            string configuration = inEditor ? "InEditor" : "Player";
```

Comment:
>"InEditor" : "Player" [](start = 46, length = 21)

it feels like these should be hard coded as constants somewhere for consistency throughout the exporters

Reply:
Possibly, but will save that for a later change.